Report.java

```diff
@@ -202,4 +205,16 @@ private void printSpriteCounts(PrintWriter printW, int i)
             printW.println();
         }
     }   
+
+    /**
+     * Method to get the current datetime for report.
+     *
+     * @return formatted datetime
+     */
+    public String getReportDateTime()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
```

Comment:
I would leave the format how you have it @haietza .  While it may not be the easiest to read I think we need the granularity of the HHmmss.